packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -502,7 +502,9 @@ export class AppStudioPluginImpl {
       }
     } else {
       appDirectory = `${ctx.root}/.${ConfigFolderName}`;
-      manifestString = JSON.stringify(ctx.app);
+      manifestString = (
+        await fs.readFile(`${ctx.root}/.${ConfigFolderName}/${REMOTE_MANIFEST}`)
```

Comment:
for azure project, we need to generate remote manifest, not just read manifest file.